### DISCUSSION TITLE
Running Judi in Docker

### DISCUSSION THREAD

**Discussion by GuillaumeSRichard** [slimgroup/JUDI.jl#75] [Q&A] [answered]
Hello

I have been using Judi on Docker, but had some trouble identifying the best image to use:

- Image philippwitte/judi:v1.5 works well but is now 2 years old.
- Image mloubout/judi-base:1.0 seems to be missing most of the dependencies necessary to run the basic examples.
- Images mloubout/judi-cpu:1.4.3 and mloubout/judi-gpu:1.0  won't run on  my system (the container dies immediately with an exit code 0 on Docker Desktop). 

Any pointer would be helpful. Thanks.

Guillaume

**Comment by mloubout** [comment]
Hi

Thanks for reaching out. Let me check the images again and fix it if necessary. 

I agree that ` philippwitte/judi:v1.5` is now quite old, so I have updated the README and will make sure the new image (mloubout) runs correctly.

>most of the dependencies necessary to run the basic examples.

You mean the additional julia packages?

**Reply by GuillaumeSRichard** [reply]
Yes, the additional Julia Packages.

**Comment by mloubout** [comment]
>mloubout/judi-cpu:1.4.3 and mloubout/judi-gpu:1.0 

These two images were originally made to run on Azure batch. I will double check what local issues it may create and create new images if necessary. Thanks for the feedback.

**Comment by mloubout** [comment, marked as answer]
I just pushed a new version `mloubout/judi-base:1.1` that should have all the dependencies installed for the examples.

**Reply by GuillaumeSRichard** [reply]
Thanks! The new version mloubout/judi-base:1.1 seems to work very well as far as I could test.

**Reply by GuillaumeSRichard** [reply]
And I assume the docker file corresponding to mloubout/judi-base:latest is and will remain for the future: https://github.com/slimgroup/JUDI.jl/blob/master/docker/Dockerfile
Correct?

**Reply by mloubout** [reply]
Yes it is. THe two other dockerfile in that repository are the one used for the `judi-cpu` and `judi-gpu` images for azure.
